docs/en/lua-developer-faq.md

```diff
@@ -1,12 +1,28 @@
 # FAQ
 
-**# # # Work in Progress # # #**
+*This FAQ was started by [Terry Ellison](https://github.com/TerryE) as an unofficial FAQ in mid 2015.  This version as at April 2017 includes some significant rewrites.*
 
-*This was started by [Terry Ellison](https://github.com/TerryE) as an unofficial FAQ in mid 2015. It never became officially official and it is in need of an overhaul, see [#937](https://github.com/nodemcu/nodemcu-firmware/issues/937). Yet, it is still very valuable and is, therefore, included here.*
- 
 ## What is this FAQ for?
 
-This FAQ does not aim to help you to learn to program or even how to program in Lua.  There are plenty of resources on the Internet for this, some of which are listed in [Where to start](#where-to-start).  What this FAQ does is to answer some of the common questions that a competent Lua developer would ask in learning how to develop Lua applications for the ESP8266 based boards running the [NodeMcu](http://NodeMCU.com/index_en.html) firmware.
+This FAQ does not aim to help you to learn to program or even how to program in Lua.  There are plenty of resources on the Internet for this, some of which are listed in [Where to start](#where-to-start).  What this FAQ does is to answer some of the common questions that a competent Lua developer would ask in learning how to develop Lua applications for the ESP8266 based boards running the [NodeMcu firmware](https://github.com/nodemcu/nodemcu-firmware).  This includes the NodeMCU Devkits.  However, the scope of the firmware is far wider than this as it can be used on any ESP8266 module.
+
+## What has changed since the first version?
```

Comment:
You have a point here - more than one actually.

> I don't know who owns the NodeMCU project Facebook page

"We" do. I don't know who else has write access to it but at least the originators and I do.

> but Facebook is closed resource and many of us may not even have a Facebook account 

True, I keep forgetting that (what a shame it is closed btw). I don't have a personal account either and I personally couldn't care less about FB but I just accept that it's a useful medium for a project like ours.

You're right about WP as well.

I'll be happy to create a "Changes in NodeMCU" section and transfer your chapter to it. It'll be  on the [README](https://github.com/nodemcu/nodemcu-firmware/blob/dev/README.md), though. The space between the "Releases" and "Support" sections looks appropriate to me.